ihatemoney/tests/general_test.py

```diff
@@ -0,0 +1,271 @@
+import io
```

Comment:
I'm not sure about the "general" name here, but that might just be me. What about `main_test`?

Reply:
Sure, i'll do it :)